plugins/uefi/fu-plugin-uefi.c

```diff
@@ -58,6 +58,9 @@ fu_plugin_init (FuPlugin *plugin)
 	fu_plugin_add_rule (plugin, FU_PLUGIN_RULE_RUN_AFTER, "upower");
 	fu_plugin_add_report_metadata (plugin, "FwupdateVersion", LIBFWUP_LIBRARY_VERSION);
 	fu_plugin_add_report_metadata (plugin, "EfivarVersion", EFIVAR_LIBRARY_VERSION);
+#if LIBFWUP_CHECK_VERSION(12,0)
```

Comment:
This is unfortunately going to fail if compiled with fwupdate 10 or less.  It's fixed in fwupdate 11 (https://github.com/rhboot/fwupdate/commit/b7d2f634969b78b9084cb06781d92a4d3fb8530e).  So you'll probably need to add some magic in `meson.build` until the minimum version dependency for fwupd is at least 11.

Reply:
Ohh, I forgot about that, thanks.